sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1744,6 +1744,17 @@ object SQLConf {
          "and from_utc_timestamp() functions.")
     .booleanConf
     .createWithDefault(false)
+
+  private[sql]
+  val CONF_SOURCES_BINARY_FILE_MAX_LENGTH = "spark.sql.sources.binaryFile.maxLength"
+  private[sql]
+  val SOURCES_BINARY_FILE_MAX_LENGTH = buildConf(CONF_SOURCES_BINARY_FILE_MAX_LENGTH)
```

Comment:
Nit: I think we can follow the other SQLConf here by putting the conf key into `buildConf` without assigning it into a variable. Also, we can remove the `private[sql]`.

```
val SOURCES_BINARY_FILE_MAX_LENGTH = buildConf("spark.sql.sources.binaryFile.maxLength")...`
```

We can set the key with `SQLConf.SOURCES_BINARY_FILE_MAX_LENGTH.key`.

Reply:
Done.